slices/main/assets/js/app.js

```diff
@@ -1,3 +1,4 @@
+import "../../../../app/assets/builds/tailwind.css";
```

Comment:
Can't say I particularly love all the `../`’s here, but I’m not sure if there’s any better option for a base shared Tailwind file like this.

Reply:
We could use TypeScript which would let us use `tsconfig.json` to set a path alias perhaps?

Reply:
That's worth a try, thanks @makenosound! Adding TypeScript is on my list here, so I'll take another look once that's in place.